### DISCUSSION TITLE
Error: error:0909006C:PEM routines:get_name:no start line     at Sign.sign (node:internal/crypto/sig:131:29)

### DISCUSSION THREAD

**Discussion by kaxada** [octokit/octokit.js#2269] [Q&A] [answered]
I built a `node:16-alpine` docker image of my GitHub app that is created with the `octokit library` and deployed it to a `ubuntu 20.04` digital ocean droplet. I supplied the `envvars` by placing them in the `/bash_profile` file of my droplet. 

When a webhook is received, the response fails due to an authentication error that may be related to a privateKey issue. the error is;
```sh
node:internal/crypto/sig:131
  const ret = this[kHandle].sign(data, format, type, passphrase, rsaPadding,
                            ^

Error: error:0909006C:PEM routines:get_name:no start line
```

PS: my privatekey envvar is supplied in a format `privateKey: "-----BEGIN RSA PRIVATE KEY-----\nMIIE...w==\n-----END RSA PRIVATE KEY-----"`

what could be causing that auth error?

**Comment by gr2m** [comment, marked as answer]
I think I saw this error before when the private key was not formatted correctly. And I remember that was a problem with a specific Node 16 version, not too long ago. Downgrading to a lower Node 16 version fixed the problem. Hope that helps
